vcs/src/main/java/org/openjdk/skara/vcs/git/GitRepository.java

```diff
@@ -594,6 +594,9 @@ public void push(Hash hash, URI uri, String ref, boolean force, boolean includeT
 
         if (includeTags) {
             cmd.add("--tags");
+            if (force) {
+                cmd.add("--force");
```

Comment:
Is there a reason not to ignore the `force` flag when not pushing tags? It would seem more consistent to always look at the `force` flag and add `--force` is set. That way a repo configured not to push tags would still be able to handle a source branch that was force-pushed.

Reply:
I was torn about this. I went this way to change as little current behavior as possible, mainly because I don't fully understand the nuances of git behavior here. Note that further down, `force==true` is already handled for branches by prepending `+` to the refs. On the other hand, the current proposal may be inconsistent, which is also bad long term.

Perhaps the best solution would be to get rid of the `+` below and just add `--force` when `force==true`?

Reply:
Using `--force` consistently sounds cleanest, but if you want to limit the scope of this PR and do that in a follow-up PR, that would be good.